.github/workflows/run-tests.yml

```diff
@@ -10,20 +10,29 @@ permissions:
   contents: read
 
 jobs:
+  version:
+    runs-on: ubuntu-18.04
+    name: Parse versions
+    outputs:
+      node-version: ${{ steps.parse-node-version.outputs.nvmrc }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+      - name: Parse Node version
+        id: parse-node-version
+        run: echo "##[set-output name=nvmrc;]$(cat .nvmrc)" 
   install-and-compile:
     runs-on: ubuntu-18.04
     name: Install and compile
+    needs: version
```

Comment:
Looks good to me! Can we back-port this approach to pay-frontend?

Reply:
Linked https://github.com/alphagov/pay-frontend/pull/2745